tests/installation/grub_test.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "basetest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    up_and_down();
```

Comment:
after the recent change by maxlin (press 'up' once) seems to be just enough. Is this really still necessary? We currently seem to have a situation where workers struggle with heavy load but have no problem to check the screen at least once during the grub timeout to abort the timeout after one successful screen match. I prefer to keep the current approach